receiver/syslogreceiver/go.mod

```diff
@@ -37,11 +37,11 @@ require (
 	go.uber.org/atomic v1.10.0 // indirect
 	go.uber.org/multierr v1.8.0 // indirect
 	go.uber.org/zap v1.23.0 // indirect
-	golang.org/x/net v0.0.0-20220722155237-a158d28d115b // indirect
+	golang.org/x/net v0.0.0-20220812174116-3211cb980234 // indirect
```

Comment:
How is this related to the promtail receiver?

Reply:
this is how `make gotidy` works. I added loki dependencies, and they need higher versions of `golang.org/x/net` and `google.golang.org/genproto`. That's why components that have dependency on `golang.org/x/net` and `google.golang.org/genproto` were updated

Reply:
I don't think that's the case, for example `make gotidy` did not bump the `golang.org/x/net` dependency on `internal/kubelet`: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/c10532dc4e76de3a703fb485afcffae663dc7b8c/internal/kubelet/go.mod#L40 and the older version remains in your branch if I didn't misread anything.

I think what happened is that for some reason the `pkg/stanza` go.mod got modified (maybe you did it manually or maybe something else triggered it), which caused these log-related receivers to have their go.mod modified. If you don't have a good idea as to why `pkg/stanza` got modified, I would suggest to try and revert the changes, they are not strictly related to the promtail receiver and I wouldn't want to run the risk of introducing unintended changes.

Reply:
Awesome :)